test/asn1_internal_test.c

```diff
@@ -70,7 +70,7 @@ static int test_standard_methods()
 
     for (tmp = standard_methods, i = 0; i < OSSL_NELEM(standard_methods);
          i++, tmp++) {
-        if ((*tmp)->pkey_id < last_pkey_id) {
+        if (!TEST_int_ge((*tmp)->pkey_id, last_pkey_id)) {
```

Comment:
I'm not quite sure what this test does but it doesn't seem to be exercising a failure here, so I think this should stay as is.